components/social-links/social-links.test.js

```diff
@@ -24,6 +24,7 @@ describe('the Link component', () => {
         { href: 'https://www.facebook.com/', text: 'facebook' },
         { href: 'https://www.google.com/', text: 'google' },
       ]}
+      name="name"
     />);
     const links = screen.getAllByRole('link');
     expect(links[0]).toHaveAttribute('href', 'https://www.facebook.com/');
```

Comment:
@changangus since we moved the logic to format the `ariaLabel` prop to this component, I think it's appropriate to test that formatting here. I suggest that we split the current `renders links given an href and text in the correct order` test into two tests:
- `renders multiple social links in the correct order`: use the current test data with 2 social links. the test will assert that `screen.getAllByRole('link')` has a length of 2, and that the text content of each link is correct. No need to test other link attributes in this test. 
- `renders a social link`: the test data will include a single social link, the test will assert that the link contains the href, text, and _formatted_ aria label.